stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -42,7 +51,7 @@ public data class Attachment(
 
     @IgnoreSerialisation
     @IgnoreDeserialisation
```

Comment:
It's kind of black magic how it gets obtained from BE :-(

Reply:
It definitely is, but can we do anything better? There can be arbitrary dynamic data in the JSON we send/receive.